saleor/checkout/models.py

```diff
@@ -301,3 +300,9 @@ def __setstate__(self, data):
     def is_shipping_required(self) -> bool:
         """Return `True` if the related product variant requires shipping."""
         return self.variant.is_shipping_required()
+
+
+class CheckoutMetadata(ModelWithMetadata):
```

Comment:
A comment would be useful to explain why in this case metadata is moved to a separate model.